src/code/views/share-dialog-view.js

```diff
@@ -83,15 +83,11 @@ export default createReactClass({
     const shared = client.isShared()
     const sharedDocumentUrl = client.state?.currentContent?.get("sharedDocumentUrl")
     const sharedDocumentId = this.getSharedDocumentId()
-    const useInternalLink = true
     if (shared) {
       if(sharedDocumentUrl) {
-        return `${this.props.client.getCurrentUrl()}#shared=${encodeURI(sharedDocumentUrl)}`
+        return sharedDocumentUrl
```

Comment:
I think this approach breaks the normal share links. I tried it here: https://codap.concord.org/releases/latest/static/dg/en/cert/index.html?cfmBaseUrl=https://cloud-file-manager.concord.org/branch/simplify-share-urls/js
When I share a document and I look at the 'Link' tab of the sharing dialog I see a URL of:
https://cfm-shared.concord.org/NcSu07SspsIyKNpZE5Ge/file.json
That link won't work if someone sends it to someone else.

So I think what is needed is that the getShareLink needs to stay mostly the same. And the code in `getLARA()` needs to change so it uses just the sharedDocumentUrl